### ISSUE TITLE
Screen Reader: Many Pages: Various buttons are announce only option, no visible name

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74877]
## Action Performed
Prerequisites:
- The user is signed in

1. Using JAWS + Chrome, open the site https://new.expensify.com/settings/profile
2. Press Tab key to navigate through the profile menu buttons.
3. Note how the elements are displayed and announced.

Other Occurrences:
1. On Settings - Preferences: the buttons below switches
2. On Workspaces - Overview: the buttons below Edit Photo
3. On Workspace - Reports: the button 'Detailed report title'
4. On Workspace - Categories - Settings
5. On Workspace - Workflows
6. On Workspace - Workflows - Edit Approval Workflow
7. On Workspace - Rules
8. On Workspace - Expensify Card - Confirm currency and country
9. On Workspace - Invoices
10. On Workspace - Create Workspace - Confirm Workspace
11. On Create Report - Add payment card
12. On Track Distance
13. On Send Invoice
14. On Create Expense flow
15. On Paid Expense details flow
16. On Reports flow

## Expected Result
Each button should announce its visible name together with the selected option, e.g.:
- 'display name <option>'

## Actual Result
Various buttons are announced without their visible names; only the selected option can be heard.

For example, a user sets their display name to 'Name Last', Screen reader then announces the button as 'Name Last' The dynamic change may be confusing for users who are expecting the same button name, or who may think this is a new button option presented

HTML:
<div aria-label="John Smith" role="menuitem" data-tag="pressable" tabindex="0" class="css-g5y9jx r-1otgn73 r-1loqt21 r-17gur6a r-18u37iz r-1wtj0ep r-1mmae3n r-o4dypm r-1fkl15p r-iyfy8q" data-component="GenericPressable" data-element="Pressable" data-source-file="BaseGenericPressable.tsx"><div class="css-g5y9jx r-13awgt0"><div class="css-g5y9jx r-18u37iz"><div class="css-g5y9jx r-eqz5dr r-13awgt0"><div class="css-g5y9jx r-18u37iz r-105ug2t"><div class="css-g5y9jx r-1777fci r-13awgt0" style="min-height: 40px;"><div dir="auto" class="css-146c3p1 r-8akbws r-krxsd3 r-dnmrzs r-1qsk4np r-1udbk01" style="-webkit-line-clamp: 2; color: rgb(175, 187, 176); font-size: 13px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; margin-bottom: 4px; line-height: 16px; word-break: break-word;">Display name</div><div class="css-g5y9jx r-18u37iz r-1awozwy r-dnmrzs"><div dir="auto" data-selection-scrapper-hidden-element="false" class="css-146c3p1 r-1udh08x r-1udbk01 r-1iln25a" style="color: rgb(231, 236, 233); font-size: 15px; text-align: left; font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; flex-shrink: 1; white-space: pre; direction: ltr; max-width: 100%; line-height: 20px;">John Smith</div></div></div></div></div><div class="css-g5y9jx r-18u37iz" style="min-height: 40px;"><div class="css-g5y9jx r-105ug2t" style="justify-content: center; align-items: center; width: 40px;"><div class="css-g5y9jx" data-testid=""><svg xmlns="[http://www.w3.org/2000/svg"](http://www.w3.org/2000/svg%22) xml:space="preserve" viewBox="0 0 20 20" width="20" height="20" hovered="false" pressed="false" fill="#8B9C8F"><path d="M5.6 18.4c.8.8 2 .8 2.8 0l8.4-8.4-8.4-8.4c-.8-.8-2-.8-2.8 0s-.8 2 0 2.8l5.6 5.6-5.6 5.6c-.8.8-.8 2 0 2.8" style="fill-rule: evenodd; clip-rule: evenodd;"></path></svg></div></div></div></div></div></div>

## Area issue was found
Settings–Profile

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Update the aria-label to include both - the visible name and entered option, e.g.:
- 'display name <option>'
https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA14

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
https://github.com/user-attachments/assets/b7b37cf5-d571-48a0-9789-3243bb556df2

<img width="1918" height="997" alt="Image" src="https://github.com/user-attachments/assets/72d1762b-5b5b-4126-8ad4-4fb0ad4bf151" />

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** Yes
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation YES
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6985944
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by mallenexpensify** [comment]
Accessibility, unassigning, will be addressed in bulk once all issues are created.
